cmd/nerdctl/container_create.go

```diff
@@ -356,6 +356,10 @@ func processContainerCreateOptions(cmd *cobra.Command) (opt types.ContainerCreat
 	if err != nil {
 		return
 	}
+	opt.LogShim, err = cmd.Flags().GetString("log-shim")
```

Comment:
You need add the flag here https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/container_run.go#L75 first

Reply:
- I feel the CLI should be like `--log-driver uri --log-opt uri=<uri>`, or `--log-driver binary --log-opt binary=/usr/local/bin/...`
- Please update https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md
- Please add tests
- Please consider making this experimental https://github.com/containerd/nerdctl/blob/8af72334f0e73b01a9c85a746c15ac0e5772aad7/pkg/cmd/image/convert.go#L214 https://github.com/containerd/nerdctl/blob/main/docs/experimental.md